packages/api/src/controllers/asset.ts

```diff
@@ -287,6 +287,24 @@ async function validateAssetPlaybackPolicy(
         `webhook ${playbackPolicy.webhookId} not found`
       );
     }
+    const allowedOrigins = playbackPolicy?.allowedOrigins;
+    if (allowedOrigins) {
+      if (allowedOrigins.length > 0) {
+        const allowedOriginsValid = allowedOrigins.every((origin) => {
+          const url = new URL(origin);
```

Comment:
You might need to try catch this to throw a friendly error to the user. Or does it never explode here? Maybe this present is very lenient to invalid input, gpt told me that once lol